### PR TITLE
"You either die a hero or live long enough to see yourself contribute dad jokes to a GitHub repository"

### PR DESCRIPTION
**Pull request by bapspatil**

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -4,6 +4,12 @@ submit your own, if they make me laugh I'll merge them.
 
 ---
 
+**Q:** How did the developer announce his engagement?
```
Comment (styfle) [inline readme.md]:
```suggestion
**Q:** How did the developer announce their engagement?
```

File: readme.md [hunk 2]
```diff
@@ -4,6 +4,12 @@ submit your own, if they make me laugh I'll merge them.
...
+
+**A:** She `return`ed `true`!
```
Comment (styfle) [inline readme.md]:
```suggestion
**A:** They `return`ed `true`!
```

**Comment by bapspatil** [comment]
I've merged your suggestions! 😄 @styfle

**Comment by bapspatil** [comment]
`return "Welcome!";`

**Pull request merged**